packages/core/src/collider/ABoxCollider.ts

```diff
@@ -103,4 +107,18 @@ export class ABoxCollider extends Collider {
 
     // return this._corners;
   }
+
+  _raycast(ray: Ray, hit: HitResult): boolean {
```

Comment:
add @internal comments for _XX() func, otherwise it will be exposed in the document.